test/cpp/api/modules.cpp

```diff
@@ -1258,6 +1258,25 @@ TEST_F(ModulesTest, Softplus) {
   }
 }
 
+TEST_F(ModulesTest, Softshrink) {
+  const auto size = 3;
+  for (const auto lambda : {0.0, 0.42, 1.0, 4.2, 42.42}) {
+    Softshrink model {SoftshrinkOptions(lambda)};
```

Comment:
We can change this to
```suggestion
    Softshrink model {/*lambda=*/lambda};
```
to also test `SoftshrinkOptions`'s implicit constructor :D